docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,533 @@
+    "visualize(image_path, label=label, prediction=prediction, class_names=class_names, overlay=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1549af39",
+   "metadata": {},
+   "source": [
+    "The birthday party in this image should have had individual bounding boxes around each kid (the COCO guidelines state only groups with 10+ objects of the same type can be a \\\"crowd\\\" bounded by a single box). \n",
+    "\n",
+    "All of these examples received low label quality scores reflecting their low annotation quality in the original dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03d5a521",
+   "metadata": {},
+   "source": [
+    "### Other uses of visualize\n",
+    "The `visualize()` function can also depict non-issue images, labels or predictions alone, or just the image itself. Let's explore this with a few images in our dataset.\n",
+    "\n",
+    "We can save a visualization to file via the `save_path` argument. Note the label quality score is high for this example and it is marked as a non-issue. The given and predicted labels closely resemble each other contributing to the high score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e770d23",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_to_visualize = 0\n",
+    "image_path = IMAGE_PATH + labels[image_to_visualize]['seg_map']\n",
+    "print(image_path, '| idx', idx , '| label quality score:', scores[image_to_visualize], '| is issue:', image_to_visualize in issue_idx)\n",
+    "visualize(image_path, label=labels[image_to_visualize], prediction=predictions[image_to_visualize], class_names=class_names, save_path='./example_image.png')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c9464e8",
+   "metadata": {},
+   "source": [
+    "For the next example, notice how we are only passing in the given labels to visualize. We can limit visualization to either labels, predictions, or neither."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "57e84a27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_to_visualize = 1\n",
+    "image_path = IMAGE_PATH + labels[image_to_visualize]['seg_map']\n",
+    "print(image_path, '| idx', idx , '| label quality score:', scores[image_to_visualize], '| is issue:', image_to_visualize in issue_idx)\n",
+    "visualize(image_path, label=labels[image_to_visualize], class_names=class_names)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8744ab9",
+   "metadata": {},
+   "source": [
+    "For completeness, let's just look at an image alone."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0302818a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_to_visualize = 2\n",
+    "image_path = IMAGE_PATH + labels[image_to_visualize]['seg_map']\n",
+    "visualize(image_path)"
```

Comment:
Each image above has an accompanying print statement but not this one, should we add for consistency?